Source/Particles/Deposition/CurrentDeposition.H

```diff
@@ -176,4 +176,280 @@ void doDepositionShapeN(const Real * const xp, const Real * const yp, const Real
         );
 }
 
+// Compute shape factor and return index of leftmost cell where 
+// particle writes.
+// Specialized templates are defined below for orders 1, 2 and 3.
+template <int depos_order>
+AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE
+int compute_shifted_shape_factor (Real* const sx, const Real x_old, const int i_new) {return 0;};
```

Comment:
the best think is probably best to keep this unimplemented, which will avoid returning invalid results for unspecialized `depos_order`s
```suggestion
int compute_shifted_shape_factor (Real* const sx, const Real x_old, const int i_new);
```

but we have to check this compiles.

Reply:
I pushed a commit that updates this and fixes CI